rocketpy/simulation/flight.py

```diff
@@ -1671,6 +1671,7 @@ def u_dot_generalized(
         v = Vector([vx, vy, vz])  # CDM velocity vector
         e = [e0, e1, e2, e3]  # Euler parameters/quaternions
         w = Vector([omega1, omega2, omega3])  # Angular velocity vector
+        omega = [omega1, omega2, omega3]
```

Comment:
We already have `w` here, no need to create the `omega` list.

`Vector`s can be accessed just like lists with brackets (e.g. `[0]`), so using it shouldn't break anything